### DISCUSSION TITLE
Initial user stories

### DISCUSSION THREAD

**Discussion by naufaldi** [bearmentor/bearmentor-com-legacy#2] [Ideas]
add user story for discuss about Landingpage and Discovery People.

**Comment by naufaldi** [comment]
[Issue Docs](https://github.com/bearmentor/bearmentor/pull/3)

**Comment by mhaidarhanif** [comment]
Nice, we've been adding the first ones to the docs folder

**Comment by bear-mentor** [comment]
@mhaidarhanif @naufaldi @zainfathoni

As we can see, the progress of the top priorities are looking good at 🐻 [Bearmentor.com](https://bearmentor.com)
